client/app/_components/ui/dropdown/SettingDropdown.tsx

```diff
@@ -0,0 +1,44 @@
+import { RxGear, RxPerson, RxExit } from 'react-icons/rx'
+import {
+  DropdownMenu,
+  DropdownMenuLabel,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from './DropdownMenu'
+
+const SettingDropdown = () => {
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger className='pl-2 focus:outline-none'>
+        <RxGear className='h-7 w-7' />
+      </DropdownMenuTrigger>
+      <DropdownMenuContent className='m-4 w-52 font-medium'>
+        <DropdownMenuLabel>My Account</DropdownMenuLabel>
```

Comment:
Maybe we don't need this Label? Because we also have items House setting and Logout buttons in this menu!

Reply:
True! I fixed it!
https://github.com/HAK2024/fairy-share/pull/55/commits/458e0226ddb1e96e4370603a49c12b4d9f2a24ed